source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,117 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v6.0. 
 
 Latest Mattermost Releases:
+- [Release v7.1 - Extended Support Release](#release-v7-1--extended-support-release)
 - [Release v7.0 - Major Release](#release-v7-0-major-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
-- [Release v6.5 - Feature Release](#release-v6-5-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.1 - [Extended Support Release](https://docs.mattermost.com/upgrade/release-definitions.html#extended-support-release-esr)
+
+**Release Day: 2022-07-16**
+
+Mattermost v7.1.0 contains low severity level security fixes. [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) to this release is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/).
+
+### Important Upgrade Notes
+ - A new configuration option ``MaxImageDecoderConcurrency`` indicates how many images can be decoded concurrently at once. The default is -1, and the value indicates the number of CPUs present. This affects the total memory consumption of the server. The maximum memory of a single image is dictated by ``MaxImageResolution * 24 bytes``. Therefore, we recommend that ``MaxImageResolution * MaxImageDecoderConcurrency * 24`` should be less than the allocated memory for image decoding.
```

Comment:
> The maximum memory of a single image is dictated by ``MaxImageResolution * 24 bytes``. Therefore, we recommend that ``MaxImageResolution * MaxImageDecoderConcurrency * 24`` should be less than the allocated memory for image decoding.

This doesn't sound right. I assume we meant bits instead of bytes as 24-bit (true color) is the most standard bit depth used for images nowadays. With that said, nothing prevents users from uploading denser formats. The current supported limit would be (N)RGBA64 so up to 64 bits (or 8 bytes) per pixel.

Reply:
Adding @agnivade to confirm.

Reply:
@jupenur Could you point me to where this allocation happens in the JPEG decoder? I am trying to understand if this in addition to the space allocated for the raw image or if it's included in it.

Reply:
Okay so I am assuming they are separate as that data there seems to be purely allocated for processing. Still unsure where the 24 is coming from exactly though but I guess it should serve as an accurate enough estimate.